server/handlerequest.go

```diff
@@ -66,9 +82,30 @@ func handleRequest(r *repo.Repo, handler Handler, jsonBytes []byte) (replyBytes
 		err = apiErr
 		reply = responses.NewError(3, "internal error "+apiErr.Error())
 	}
+
 	return encodeReply(reply)
 }
 
+func addMetrics(metrics *status.Metrics, handlerName Handler, start time.Time, errJSON error, errAPI error) {
```

Comment:
I wouldn't pass in metrics, but would rather suggest having them as global variables

Reply:
since they are instantiated for both service types: socketserver / webserver

Reply:
what about to add another label "service" and then set socketserver / webserver?
It's more prometheus-like!